.spectral.yaml

```diff
@@ -156,6 +156,17 @@ rules:
         functionOptions:
           match: "^(dev|qa|stage|prod)(,(dev|qa|stage|prod))*$"
 
+  no-slash-before-custom-method:
+  description: "Custom methods (e.g., ':applyItem') should not be preceded by a '/'."
+  message: "The path '{{path}}' contains a '/' before a custom method. Custom methods should not start with a '/'."
+  severity: error
+  given: "$.paths"
+  then:
+    field: "@key"
+    function: pattern
+    functionOptions:
+      notMatch: "^/[^/]+/:[a-zA-Z]+$"
+
```

Comment:
Risk: Since Ciprian fixes on the spec are not backported to vbranches, the FOAS release process will fail for QA, STAGING and PROD since the spectral step will find custom method with `/:`

Reply:
great point, lets wait then!